src/AddForm.js

```diff
@@ -1,22 +1,60 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import {
   collection,
   getDocs,
   query,
   onSnapshot,
   doc,
+  where,
   addDoc,
 } from '@firebase/firestore';
 import { db } from './lib/firebase.js';
 
-const userToken = 'plushy cuny idiom';
+let userToken;
+userToken = localStorage.getItem('token');
```

Comment:
I see you changed this line to get the current user token. This solves the issue of checking against the current issue—good job debugging this.